### PR TITLE
sponsors =D

### PR DESCRIPTION
**Pull request by CodyJamesCasey**
teh wootz

### CODE REVIEW

**Comment by skeswa** [comment]
Hey @CodyJamesCasey, you have a bunch of Thumbs.db files committed. You should prolly uncommit those.

**Comment by CodyJamesCasey** [comment]
Weird, they aren't on my computer... let me investigate

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,8 @@ public/vendor
 
 # Vim
 *.swp
+
+#Mystery File
+public/img/Thumbs.db
```
Comment (skeswa) [inline .gitignore]: I think that maybe just `Thumbs.db` instead of `public/img/Thumbs.db` would be better.

**Comment by samuelcouch** [comment]
Here we go again.... HALP @SriRamanujam

**Pull request merged**